core/storage/topic/gae_models.py

```diff
@@ -232,6 +234,9 @@ class TopicSummaryModel(base_models.BaseModel):
     # The description of the topic.
     description = ndb.TextProperty(indexed=False)
 
+    # The category this topic belongs to.
+    category = ndb.StringProperty(required=True, indexed=True)
```

Comment:
Please check the "Files Changed" tab and proofread before you submit a PR for review.

Reply:
What's the problem here?

Reply:
Ah, my apologies, this was my bad. I saw the same line repeated twice and thought it was for the same model. Please ignore the previous comment -- sorry!